src/api/submission.api.ts

```
@@ -0,0 +1,24 @@
+const submissionApi = {
+  getAll() {
+    return axiosClient.get(baseUrl);
+  },
+  submit(payload: FormData) {
+    {
+      return axiosClient.post(baseUrl, payload, {
+        headers: {
+          "Content-Type": "multipart/form-data",
+        },
+      });
+    }   
```

Biome

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)